### DISCUSSION TITLE
Please read - Where should I post?

### DISCUSSION THREAD

**Discussion by briangow** [MIT-LCP/mimic-code#1087] [Announcements]
Please read this to determine the best place to post your question.  FIrst, you should decide whether to post here in Discussions or instead in [Issues](https://github.com/MIT-LCP/mimic-code/issues). Please also review the guidance at https://mimic.mit.edu/docs/help/ before posting.

Post in Discussions when:
- You're wondering how something works in MIMIC.
- You have a question about the data in MIMIC.

Post in Issues when:
- You notice a potential issue with the code or data.
- You have a suggestion for improvements.

If you decide to post in Discussions, please try to choose the most appropriate category to post under:

- **MIMIC-General**: please post here if your question relates to a topic which applies across all versions of MIMIC.  For example, if your question is about getting credentialed access on PhysioNet.
- **MIMIC-III**: please post here if your question is specific to MIMIC-III. For example, if you have a question about the _callout_ table.
- **MIMIC-IV**: please post here if your question is specific to MIMIC-IV. For example, if you have a question about the _poe_ table. 
- **MIMIC-IV-CXR**: please post here if your question is specific to MIMIC-IV-CXR. For example, if you have a question about downloading the CXR data.
- **MIMIC-IV-ED**: please post here if your question is specific to MIMIC-IV-ED. For example, if you want to understand how to determine if an ED patient was also admitted to the ICU.

**Comment by tohidzamani** [comment]
Hi,

I hope you are doing well. Thank you very much for your excellent work on the MIMIC data.

I have a question regarding injury timing. I am interested in estimating the time interval between the injury (e.g., incident or fall) and arrival at the emergency department or hospital. Could you please advise on whether information about the time of the incident or injury is available in MIMIC-IV, and if so, how it can be extracted?
